src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -850,8 +849,24 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
             });
     }
 
-    deleteBetrokkene(): void {
-        // TODO #1325
+    deleteBetrokkene(betrokkene: ZaakBetrokkene): void {
+        this.websocketService.suspendListener(this.zaakRollenListener);
```

Comment:
Moet hier niet ook de ZaakListener worden gesuspend? (this.websocketService.suspendListener(this.zaakListener);). Dat heb ik in ieder geval gedaan bij de method adresGeselecteerd. Maar misschien is dat niet goed. Overleg anders even.

Reply:
Lijkt me niet. Ik verwacht geen zaak updated van open notificaties als je de rollen aanpast. Daar is nu juist die zaak-rollen aangepast notificatie voor.